sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/FormTrainingAsyncClient.java

```diff
@@ -324,6 +329,117 @@ public FormRecognizerServiceVersion getServiceVersion() {
+     *
+     * @return A {@link PollerFlux} that polls the copy model operation until it has completed, has failed,
+     * or has been cancelled.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public PollerFlux<OperationResult, CustomFormModelInfo> beginCopyModel(String modelId,
+        CopyAuthorization target) {
+        return beginCopyModel(modelId, target, null);
+    }
+
+    /**
+     * Copy a custom model stored in this resource (the source) to the user specified target Form Recognizer resource.
+     *
+     * <p>This should be called with the source Form Recognizer resource (with the model that is intended to be copied).
+     * The target parameter should be supplied from the target resource's output from
+     * {@link FormTrainingAsyncClient#getCopyAuthorization(String, String)} method.
+     * </p>
+     *
+     * <p>The service does not support cancellation of the long running operation and returns with an
+     * error message indicating absence of cancellation support.</p>
+     *
+     * <p><strong>Code sample</strong></p>
+     * {@codesnippet com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.beginCopyModel#string-copyAuthorization-Duration}
+     *
+     * @param modelId Model identifier of the model to copy to the target Form Recognizer resource
+     * @param target the copy authorization to the target Form Recognizer resource
+     * @param pollInterval Duration between each poll for the operation status. If none is specified, a default of
+     * 5 seconds is used.
+     *
+     * @return A {@link PollerFlux} that polls the copy model operation until it has completed, has failed,
+     * or has been cancelled.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public PollerFlux<OperationResult, CustomFormModelInfo> beginCopyModel(String modelId,
+        CopyAuthorization target, Duration pollInterval) {
+        final Duration interval = pollInterval != null ? pollInterval : DEFAULT_DURATION;
+        return new PollerFlux<OperationResult, CustomFormModelInfo>(
+            interval,
+            getCopyActivationOperation(modelId, target),
+            createCopyPollOperation(modelId),
+            (activationResponse, context) -> Mono.error(new RuntimeException("Cancellation is not supported")),
+            fetchCopyModelResultOperation(modelId));
+    }
+
+    /**
+     * Generate authorization for copying a custom model into the target Form Recognizer resource.
+     *
+     * @param resourceId Azure Resource Id of the target Form Recognizer resource where the model will be copied to.
+     * @param resourceRegion Location of the target Form Recognizer resource. A valid Azure  region name supported
+     * by Cognitive Services.
+     *
+     * <p><strong>Code sample</strong></p>
+     * {@codesnippet com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.getCopyAuthorization#string-string}
+     *
+     * @return The {@link CopyAuthorization}
```

Comment:
Mention that this is passed to beginCopyModel. Link for python example [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_training_client.py#L255)

Reply:
it could be ...